src/test/pmem2_badblock/TESTS.py

```diff
@@ -10,9 +10,10 @@
 
 
 @t.linux_only
-@t.require_ndctl(require_namespace=True)
 @t.require_admin
 @g.require_granularity(g.CACHELINE)
+@t.require_ndctl(require_namespace=True)
+@t.windows_exclude
```

Comment:
I can see that you add cases for devdaxes and cases with windows_exclude, do you have any tests where windows functionality you already created in this PR is tested?